Android/app/build.gradle

```diff
@@ -57,8 +57,8 @@ android {
         // Firebase Crashlytics requires SDK version 16.
         minSdkVersion ANDROID_MIN_SDK_VERSION as int
         targetSdkVersion ANDROID_TARGET_SDK_VERSION as int
-        versionCode 64
-        versionName "1.3.7"
+        versionCode 67
```

Comment:
I wonder if we can inject these at build time, so we don't need to change code for releases.

Reply:
You can add any code to the gradle task, for example, download the version from Play Store. But would that be more complicated?